docs/drf_yasg_integration.rst

```diff
@@ -69,4 +73,20 @@ following code to decorate your JWT ``View`` definitions.
         def post(self, request, *args, **kwargs):
             return super().post(request, *args, **kwargs)
 
+
+    class TokenBlacklistResponseSerializer(serializers.Serializer):
+        def create(self, validated_data):
+            raise NotImplementedError()
+
+        def update(self, instance, validated_data):
+            raise NotImplementedError()
+
+
+    class DecoratedTokenBlacklistView(TokenBlacklistView):
+        @swagger_auto_schema(
+            responses={
+                status.HTTP_200_OK: TokenBlacklistResponseSerializer})
```

Comment:
```suggestion
        @swagger_auto_schema(
            responses={
                status.HTTP_200_OK: TokenBlacklistResponseSerializer,
            }
        )
```

Reply:
Added line breaks. and, how can I join a team?

Reply:
Awesome, and yup you're in the team! I'll be able to ping y'all with the team mention. Thanks for helping so much!